redhat-developer/app-services-cli
# feat(consumer-group list): add search flag

Add `--search` flag for consumer group list.

Closes #714 

### Verification Steps

1. Create consumer groups for a Kafka instance.
2. Run consumer group list command with search flag.
```
go run ./cmd/rhoas kafka consumer-group list --search console-consume -o yaml
```
3. It should return the consumer groups.

### Type of change


### Checklist
